app/javascript/order_screen.js

```
@@ -354,7 +368,9 @@ document.addEventListener('turbolinks:load', () => {
             }
 
             /* eslint-disable no-undef */
+          setTimeout(() => {
```

ESLint

[error] Expected indentation of 12 spaces but found 10.

(indent)